part1/part1.md

```diff
@@ -109,6 +177,9 @@ false|false|false|                       |
   * Write an `until` loop that prints out all of the numbers that are less than 20 and are not divisible by 6.
```

Comment:
Got it.  I'll put it back.

On Thu, Mar 17, 2016 at 1:52 PM, Haseeb Qureshi <notifications@github.com>
wrote:

> In part1/part1.md
> <https://github.com/appacademy/jump-start/pull/13#discussion_r56575658>:
>
> > @@ -109,6 +177,9 @@ false|false|false|                       |
> >    * Write an `until` loop that prints out all of the numbers that are less than 20 and are not divisible by 6.
>
> Yes, but we use them a lot in the solutions, and I think they make a lot
> of code easier to parse for newbies. I'd go over them early on, especially
> for people unfamiliar with Ruby.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/appacademy/jump-start/pull/13/files/0834e5e805d7eb68486c30f480ef920ab20ed949#r56575658>
>